Atualiza valores dos formularios a partir do objeto values

quixote15 opened the pull request:
Let's Say we have some credit card in our database and we want to edit it using this card form. I was struggling with this issue and found out that when I passed the `values`  prop to the `CreditCardInput` nothing would happen. 

Also, I tried to use additionalInputsProps as suggested in the readme:

     <CreditCardInput 
        ...other props
        additionalInputsProps= name: {
        defaultValue: 'Tony Stark',
     } />


But still the values I passed were reflected in the form or cardView. So, I noticed that in the `connectToState.js` the values were only injected or passed to the `CreditCardInput` component if the user manually changed the TextInputs values. So What I did to fix this was setting the values in the `componentDidMount`:

    // src/connectToState.js
    componentDidMount = () => setTimeout(() => {   
       this.props.autoFocus && this.focus("number");
       this.setValues(this.props.values); // Added this line to initially set default values to form
    });


So now, using the component like this works fine:


    <CreditCardInput
          ...
          values={{
           name: "Tony Stark",
           number: "4111***111",
           expiry: "12/22",
           cvc: "122"
         }}


I'd like to continue using and contributing with this repo. Also, I need this change/feature in order to finish a project I'm working on...So it'd be great if you upgrade the rpm version so I don't have to use it locally. 

Nice work, keep it up!

Commit:
Atualiza valores dos formularios a partir do objeto values

yuritoledo reviewed (changes requested):
Thank you for your contrib! :)

Please, remove the binary files and add gradle files in gitignore

quixote15 commented:
Sorry, I was working on something else and pushed to the master branch. For this pull request I just modified one file.

By the way, I finished feature I that I needed for a project and I want to know if I could open another PR for that, if it interests you. I added a flag so that the user can change the direction of the credit card form to vertical/horizontal. What do you think ?

yuritoledo commented:
@quixote15 Sure, it's a good feature, if you could, open a PR.
